tests/python/frontend/tensorflow2/common.py

```diff
@@ -23,8 +23,7 @@
 
 from tvm.runtime.vm import VirtualMachine
 import tvm.contrib.graph_executor as runtime
-from tvm.relay.frontend.tensorflow import from_tensorflow
-
+from tvm.relay.frontend.tensorflow2 import from_tensorflow
```

Comment:
Previously the existing parser from `relay/frontend/tensorflow.py` was being used to test the basic TF2 ops, functions and models. From this PR, all of those test as well as the new ones are being passed through the new frontend parser for TF2, namely `relay/frontend/tensorflow2.py`